less/styles.less

```diff
@@ -51,8 +62,43 @@ body {
   .justify-content(center);
 }
 
-.hero-unit h1 {
```

Comment:
I assume so but I could be wrong.  Personally hierarchy-wise I like having one single `<h1>` elements on a page and to me all the `<h1>` hero heading should share the same styling.  We don't have to worry about this here as @gesa will be refactoring CSS/LESS later.  (We can do a design review after).